### ISSUE TITLE
This plug-in doesnt seem to be working for me

### ISSUE THREAD

**Issue by fnavas00** [Parik27/DragonFF#30]
I am brand new to "modding" GTA SA. I am trying to import the models into blender but the only files I can find are "wheels.dff, hoop.dff, arrow.dff, air_vlo.dff, zonecylb.dff, grass.dff, peds.col, vehicles.col, weapons.col" in the models folder anytime I import these they objects are overlapping and are not textured whatsoever... the peds.col file just outright doesn't work when i import it. The weapons.col file just imports like 50 sphere emptys... what am i doing wrong?

**Comment by Parik27** [comment]
The actual models that are related to peds and vehicles or map objects are stored in the gta3.img. You need to have a tool to extract these img files to get those models. 

There's a dff file for each model which stores the geometry for it. This is what the plugin can edit at the moment. You'll have to extract the txd manually to get it to texture. You can use Magic.TXD to extract these txd files. 

The .col files have only import support. These are used for collisions. The collision empties are used by the engine to optimize the collision detection. There are multiple collision files in the same .col file which is probably why you get overlapping textures. These models currently have limited export support, and if you're new to modding you'll probably not be looking into editing these just yet.

So basically, you have to extract gta3.img, and get the models and textures from there to load in Blender. After you've done editing them, you have to put them back into the img file.

Some IMG editing tools you can use:
Alci's IMG Editor - https://www.gtagarage.com/mods/show.php?id=17269
IMG Tool - https://www.thegtaplace.com/downloads/f552-img-tool-20
